src/root-package-json.js

```diff
@@ -113,15 +113,15 @@ let scripts = {
        * Colors are customizable
        */
       start: "concurrently 'npm:start:*' --restart-after 5000 --prefix-colors cyan,white,yellow",
-      'start:addon': `pnpm --filter ${addonName} start --no-watch.clearScreen`,
-      'start:tests': `pnpm --filter ${testAppName} start`,
+      'start:addon': `pnpm --filter ${addonName} start`,
+      'start:test': `pnpm --filter ${testAppName} start`,
       /**
        * Note that this test is different from v1 addon's test, which runs all of ember-try as well.
        * ember-try requires some alternate lockfile behaviors that we can't easily abstract into a
        * package.json script -- but will be present in C.I.
        *  (this is a consequence of enforced strict peers)
        */
-      test: `pnpm --filter ${testAppName} test`,
+      test: "pnpm --filter '*' test",
```

Comment:
I used `--filter '*'` to better match the pattern for `lint` and `lint:fix` scripts.